components/Common/NavBar.tsx

```diff
@@ -118,7 +128,7 @@ const NavBar: FC = () => {
               },
             })}
             maxMenuHeight={130}
-            placeholder="Search events and blogs"
+            placeholder="Search events, blogs and academics"
```

Comment:
```suggestion
            placeholder="Search for something"
```
Let's just change it to `Search for something` since it will be a long text otherwise 😅

Reply:
done